x-pack/plugin/esql/compute/src/test/java/org/elasticsearch/compute/aggregation/TopIpAggregatorFunctionTests.java

```diff
@@ -9,26 +9,13 @@
 
 import org.apache.lucene.document.InetAddressPoint;
 import org.apache.lucene.util.BytesRef;
-import org.elasticsearch.compute.data.Block;
-import org.elasticsearch.compute.data.BlockFactory;
-import org.elasticsearch.compute.data.BlockUtils;
-import org.elasticsearch.compute.operator.SequenceBytesRefBlockSourceOperator;
-import org.elasticsearch.compute.operator.SourceOperator;
 
 import java.util.List;
-import java.util.stream.IntStream;
-
-import static org.hamcrest.Matchers.contains;
-
-public class TopIpAggregatorFunctionTests extends AggregatorFunctionTestCase {
-    private static final int LIMIT = 100;
 
+public class TopIpAggregatorFunctionTests extends AbstractTopBytesRefAggregatorFunctionTests {
```

Comment:
I yanked the bytes behavior to a common class. It's tiny, but feels like it saves a bit of copy and paste and the compiler will tell you the variant bits.